src/Concerns/InteractsWithMouse.php

```diff
@@ -153,6 +156,23 @@ public function rightClick($selector = null)
         return $this;
     }
 
+    /**
+     * Control click the element at the given selector.
+     *
+     * @param  string|null  $selector
+     * @return $this
+     */
+    public function controlClick($selector = null)
+    {
+        return $this->withKeyboard(function (Keyboard $keyboard) use ($selector) {
+            $key = OperatingSystem::onMac() ? WebDriverKeys::META : WebDriverKeys::CONTROL;
```

Comment:
I don't quite follow this part. On my macOS keyboard I have a control key **and** a "meta" / command key. Why do we use the meta key for Mac here if some Mac keyboards have an actual control key?

Reply:
`COMMAND` and `META` represent the same keystroke in Mac : https://github.com/php-webdriver/php-webdriver/blob/a1578689290055586f1ee51eaf0ec9d52895bb6d/lib/WebDriverKeys.php#L90

The primary issue here is that Windows uses Control+click while Mac uses Command+click to trigger open in new tab. Similar issue can occur for action combo such as copy text etc.